### DISCUSSION TITLE
More colors in NeogitLogView

### DISCUSSION THREAD

**Discussion by LintaoAmons** [NeogitOrg/neogit#596] [Ideas]
Here's terminal `git log`

![image](https://github.com/NeogitOrg/neogit/assets/95092244/5a190dbc-2bce-4b78-8b6c-25806f439e3b)

Here's current `NeogitLogView` under my color scheme

![image](https://github.com/NeogitOrg/neogit/assets/95092244/8654a6f7-0638-45f7-a3f7-9aa40951c71c)

I think the terminal `git log` have more color than we have in `NeogitLogView`

**Comment by CKolkey** [comment]
Did you enable the `--colors` flag in the log popup?
